src/test/php/net/xp_framework/unittest/util/cmd/ConsoleTest.class.php

```diff
@@ -214,13 +214,10 @@ protected function initialize($console, $assertions) {
     Console::initialize($console);
     try {
       $assertions();
-    } catch (\unittest\AssertionFailedError $e) {
-      // finally
-    } ensure($e); {
+    } finally {
       Console::$in= $in;
```

Comment:
The code here was never 100% correct, as it would fail to reset console in, out and err streams if any exception other than `AssertionFailedError` was raised.